plugins/database/src/db_plugin.cpp

```diff
@@ -10208,34 +10208,11 @@ irods::error db_mod_access_control_op(
                 return ERROR( CAT_INVALID_ARGUMENT, "unknown collection or file" );
             }
             if ( status1 != CAT_UNKNOWN_COLLECTION ) {
-                if ( logSQL != 0 ) {
-                    log_sql::debug("chlModAccessControl SQL 12");
-                }
-                int status = cmlCheckDirOwn( _path_name,
-                                             _ctx.comm()->clientUser.userName,
-                                             _ctx.comm()->clientUser.rodsZone,
-                                             &icss );
-                if ( status < 0 ) {
-                    return ERROR( status1, "cmlCheckDirOwn failed" );
-                }
-                snprintf( collIdStr, MAX_NAME_LEN, "%d", status );
+                return ERROR( status1, fmt::format("User {} has insufficient permission to modifiy collection: {}", _ctx.comm()->clientUser.userName, _path_name) );
```

Comment:
As a convention, we normally wrap the `{}` placeholders with square brackets in case a value is an empty string.